nixos/modules/services/networking/tinydns.nix

```diff
@@ -32,23 +220,48 @@ with lib;
   config = mkIf config.services.tinydns.enable {
     environment.systemPackages = [ pkgs.djbdns ];
 
-    users.users.tinydns = {};
+    users.users.tinydns = {
```

Comment:
Is tinydns generating runtime data which a normal system user needs to access? I've never used this software, but from simply looking at this change it appears as if you could use a DynamicUser instead of creating a new static user with static ids.

See: https://github.com/NixOS/nixpkgs/issues/11908#issuecomment-254711838 and https://github.com/NixOS/nixpkgs/issues/20186#issue-187497088

Reply:
It would not be necessary in this PR, but I have an extension in the pipeline (which only makes sense to introduce after this change has been approved) that would make use of a named user for sending authenticated DNS database updates to sub-ordinate tinydns instances.

Reply:
Ah, OK.